src/Evolution/DiscontinuousGalerkin/ObserveFields.hpp

```diff
@@ -90,24 +82,16 @@ class Observe;  // IWYU pragma: keep
  * - `Error*` = errors in `AnalyticSolutionTensors` =
  *   \f$\text{value} - \text{analytic solution}\f$
  *
- * Writes reduction quantities:
- * - `Time`
- * - `NumberOfPoints` = total number of points in the domain
- * - `Error*` = errors in `AnalyticSolutionTensors` =
- *   \f$\operatorname{RMS}\left(\sqrt{\sum_{\text{independent components}}\left[
- *   \text{value} - \text{analytic solution}\right]^2}\right)\f$
- *   over all points
- *
- * \warning Currently, only one observation event can be triggered at
- * a given time.  Causing multiple events to run at once will produce
- * unpredictable results.
+ * \warning Currently, only one volume observation event can be
+ * triggered at a given time.  Causing multiple events to run at once
+ * will produce unpredictable results.
  */
 template <size_t VolumeDim, typename... Tensors,
           typename... AnalyticSolutionTensors, typename EventRegistrars,
           typename... NonSolutionTensors>
-class Observe<VolumeDim, tmpl::list<Tensors...>,
-              tmpl::list<AnalyticSolutionTensors...>, EventRegistrars,
-              tmpl::list<NonSolutionTensors...>>
+class ObserveFields<VolumeDim, tmpl::list<Tensors...>,
```

Comment:
maybe name this `ObserveFieldsAndErrors` as I imagine there will soon be an `ObserveFields` that will be used when there is not an analytic solution?